### DISCUSSION TITLE
Favorite and recommended related tracks (Post favorite and recommended related tracks here).

### DISCUSSION THREAD

**Discussion by 01GOD** [copych/AcidBox#32] [General]
The kind of music I would still likely be making if it was profitable enough to do so: 
https://music.apple.com/us/album/zenergy-electro-mech-tronica-single/1524605668
https://music.apple.com/us/album/zenergy-the-world-is-a-simulation-feat-the-buddha/1525070859?i=1525070860
https://music.apple.com/us/album/zenergy-the-bass-ship-energize-single/1525070861

(Yes, the style of bass drum in the first one is My favorite also, and I may update the others with that style drum someday. Matching production style on 2 OS's was a lot to do.)

**Comment by oracle-999** [comment]
The page can't  be found now...you mean drum&bass?
